AsyncGuidance.md

```diff
@@ -27,7 +27,7 @@ Asynchronous programming has been around for several years on the .NET platform
 in C# 5 asynchronous programming has become mainstream. Modern frameworks (like ASP.NET Core) are fully asynchronous and it's very hard to avoid the async keyword when writing
 web services. As a result, there's been lots of confusion on the best practices for async and how to use it properly. This section will try to lay out some guidance with examples of bad and good patterns of how to write asynchronous code.
 
-## Asynchrony is viral 
+## Asynchrony is vital 
```

Comment:
I'm convinced that this is **not** a typo. The 'viral' refers to the effects on the codebase when you start using the async-await keywords. It is like a 'virus' infection, it spreads through the application layers.

Reply:
hahaha. you are funny